build/worker/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM cargo.caicloud.io/caicloud/worker-base:1.11.2-dind
```

Comment:
Where is the Dockerfile of `cargo.caicloud.io/caicloud/worker-base:1.11.2-dind`?
Maybe merge Dockerfile into it is better.

Reply:
the base image is `docker:1.11.2-dind`, tested.